buildui.py

```diff
@@ -0,0 +1,16 @@
+import alqtendpy.compileui
+import pathlib
+
```

Comment:
Is the name builldui ok?  This builds the ui for both st and stlib.

Reply:
The name is ok. If I was going to suggest something else, I'd put underscores in there. `build_ui.py`. Not sure if it's worth it at this point.